README.md

```diff
@@ -7,6 +7,10 @@ This repository contains official Java libraries for Azure services. For referen
 
 You can find a complete list of all the packages for these libraries [here](packages.md).
 
+### Important: 
+We have released 'azure-sdk-for-java' GA( i.e. General Availability ) in November 2019. We recommend using stable and officially supported GA versions of these packages.
```

Comment:
How about: 

"The Azure SDK team is pleased to make available the November 2019 client library General Availability (GA) release.  We strongly recommend using the GA libraries in all production environments as these libraries are well tested and officially supported by Microsoft.  More details, including installation instructions can be found here"